scripts/zones/Dragons_Aery/mobs/Nidhogg.lua

```diff
@@ -11,6 +11,7 @@ local entity = {}
 
 entity.onMobSpawn = function(mob)
     mob:setLocalVar("[rage]timer", 3600) -- 60 minutes
+    mob:setMobMod(xi.mobMod.WEAPON_BONUS, 60) -- Level 90 + 50 = 140 Base Weapon Damage
```

Comment:
Same here.

The rest of the PRlooks good to me.

Reply:
Fixed - it was supposed to be 50